packages/yasgui/package.json

```diff
@@ -17,20 +17,22 @@
   "files": [
     "test",
     "views",
-    "plugins",
+    "dist",
     "index.js"
   ],
   "scripts": {
+    "prepare": "esbuild --minify --bundle plugins/map.js --outdir=dist --splitting --format=esm",
```

Comment:
By splitting and dynamic importing the map components we reduce the initial load